bot/tests/administration/test_add_subscription.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+
+from bot.tests.base_test import BaseTest
+
+
+class TestAddSubscriptionCommand(BaseTest):
```

Comment:
chyba brakuje testu na nieprawidlowa ilosc argow

Reply:
w zasadzie to też do wszystkich xDDD gdziekolwiek to sprawdzasz to weź też i otestuj

Reply:
ale generalnie muszę pochwalić że całkiem przemyślane test case'y gratulacje <3

Reply:
Brakuje tych testów których bot nie przechodzi XDDDD stwierdziłem że może nie będę robić testów nie do przejścia XXDDDDD tylko takie działające. Chociaż dwie komendy są tak rozejsbne że i tak musiałem dać nie działające xDD

Reply:
wdym? XD jak masz buga za duzego do naprawy w tym prze to wez moze **w drodze wyjatku** (XD) dodaj wykomentowane testy i issue zeby je odkomentowac i naprawic kod

no bo wnioskuje ze juz napisales testy ale potem je skasowales, troche szkoda

Reply:
no generalnie tak zrobiłem i tylko wiesz że tego trochę będzie bo zasadniczo błędnego inputu trochę je itp

Reply:
lepiej za duzo niz za malo, nie ma co dziadowac

Reply:
No to alternatywnie zrób listę wszystkich handlerów którym brakuje testów, a potem każdego handlera po kolei w osobnym PRze łataj. Ale musisz to spsiać bo zapomnisz na 120% XD

Reply:
Na 130% nie zapomnę bo trzeba po wszystkich handlarach przejechać bo jedne nie działają a drugie plują błędami zamiast fajnym komunikatem dla usera xD

Reply:
jak ostatecznie dodałeś wszystkie testy to go resolve